x-pack/plugins/security_solution/public/entity_analytics/components/risk_summary_flyout/risk_summary.tsx

```diff
@@ -263,7 +260,7 @@ const FlyoutRiskSummaryComponent = <T extends RiskScoreEntity>({
                 <EuiBasicTable
                   data-test-subj="risk-summary-table"
                   responsiveBreakpoint={false}
-                  columns={columns}
+                  columns={columnsArray}
```

Comment:
@tiansivive Realised I was re-defining for no reason, which I think your original comment (in previous PR) alluded to. So this makes more sense :D